wolvic/browser/vr/wvr_manager.cc

```diff
@@ -386,7 +384,8 @@ void WvrManager::OnSubmitClientMojoConnectionError() {
 // for reference
 static void PopulateProfiles(
     device::mojom::XRInputSourceStatePtr& input_source,
-    const mozilla::gfx::VRControllerState& controller) {
+    const mozilla::gfx::VRControllerState& controller,
+    bool add_hand_profile) {
```

Comment:
I don't think we need to make this optional. It should be safe to always include it.

Reply:
Well perhaps is safer not to include it as you've done. It should work I think because WebXR apps should look for that profile only when they detect that hand tracking is used, but maybe they do the other way around, so let's keep this as is.